### PR TITLE
Changed text for several Activity titles

### PR DESCRIPTION
**Pull request by YaronFruchtmann**
Fixes #

## Proposed Changes

  - Fixed Titles to be different and more generic than Content
  -
  -

### CODE REVIEW

File: Insights/InputEntity_Account/AccountsActions.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ Items:
+ - Id: d6d08c94-455f-4ea5-8f76-fc6c0c442cfa
+   Title: "The user has created an account"
+   Content: "The user has created the account {{Target_Name}} {{Count}} time(s)"
+   Description: "This activity displays accounts created by the user"
+   QueryDefinitions:
+    Filter: "where OperationName in~ ('Add user', '4720')"
+    SummarizeBy: "Target_Name, Target_NTDomain, Target_UPNSuffix"
+ - Id: e0459780-ac9d-4b72-8bd4-fecf6b46a0a1
+   Title: "The user has deleted account an account"
```
Comment (shainw) [inline Insights/InputEntity_Account/AccountsActions.yaml:60]: Should be - "The user has deleted an account"

File: Insights/InputEntity_Account/AccountsActions.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    };
+    GetAccountActions('{{Account_Name}}', '{{Account_NTDomain}}', '{{Account_UPNSuffix}}', '{{Account_AADUserId}}', '{{Account_SID}}')
+    
+Activities:
+ EnabledByDefault: true
+ Items:
+ - Id: d6d08c94-455f-4ea5-8f76-fc6c0c442cfa
+   Title: "The user has created an account"
+   Content: "The user has created the account {{Target_Name}} {{Count}} time(s)"
+   Description: "This activity displays accounts created by the user"
```
Comment (shainw) [inline Insights/InputEntity_Account/AccountsActions.yaml:55]: May want to change to - "This activity displays account creation events performed by the user"

File: Insights/InputEntity_Account/AccountsActions.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+   Title: "The user has deleted account an account"
+   Content: "The user has deleted the account {{Target_Name}} {{Count}} time(s)"
+   Description: "This activity displays account deletion events performed by the user"
+   QueryDefinitions:
+    Filter: "where OperationName in~ ('Delete user', '4726')"
+    SummarizeBy: "Target_Name, Target_NTDomain, Target_UPNSuffix"
+ - Id: ad1f4269-5418-4c46-a3b6-4ec01031de60
+   Title: "The user has reset an account's password"
+   Content: "The password for account {{Target_Name}} was reset by the user {{Count}} time(s)"
+   Description: "This activity displays password reset event performed by the user"
```
Comment (shainw) [inline Insights/InputEntity_Account/AccountsActions.yaml:69]: Add and 's' to event - "This activity displays password reset event**s** performed by the user"

**Review by shainw — CHANGES_REQUESTED** [review]

**Review by RoundNutz4 — APPROVED** [review]

File: Insights/InputEntity_Account/AzActivity_AzureOps.yaml [hunk 4]
```diff
@@ -10,15 +10,14 @@ RequiredInputFieldsSets:
...
-      | where OperationName contains "Run Command on Virtual Machine"
+  AzureActivity 
+  | where (isnotempty(Account_AadUserId) and Caller =~ Account_AadUserId) or Caller =~ upn
+  | where OperationName contains "Run Command on Virtual Machine"
                or (OperationName == "List Storage Account Keys" and ActivityStatus == "Succeeded")
                or OperationName == "Create or Update Virtual Machine" 
                or OperationName == "Create Deployment"
                or OperationName == "Create role assignment"
-  | project-rename Target_AzureResource_ResourceId = ResourceId, Source_IP_Address = CallerIpAddress
+  | project-rename Target_AzureResource_ResourceId = _ResourceId, Source_IP_Address = CallerIpAddress
```
[the remark this answers is not in the document]
Comment (YaronFruchtmann) [reply Insights/InputEntity_Account/AzActivity_AzureOps.yaml:20]: So we need to see why it failed the UT. In any case, it is better this way

File: Insights/InputEntity_Account/GroupAdditions.yaml [hunk 5]
```diff
@@ -141,23 +141,23 @@ Activities:
...
+   Content: "The user has added an account to the privileged group, {{TargetDomainName}}{{TargetUserName}}, {{Count}} time(s)"
+   Description: "This activity displays the user that added an account and the account that was added to a privileged group"
    QueryDefinitions:
     Filter: "where ((MemberNameMatch == true and MemberNTDomainMatch == true) or MemberSidMatch == true) and (TargetSid matches regex WellKnownLocalSID or TargetSid matches regex WellKnownGroupSID) | where TargetSid !in ('S-1-5-32-555')"
     SummarizeBy: "SubjectAccount, TargetUserName, TargetSid"
  - Id: 5ae2baf4-de7b-40f0-a861-8852266bfcd0
-   Title: "The account was added to a remote desktop group"
-   Content: "The user has added an account to a  remote desktop group {{Count}} time(s)"
+   Title: "The user has added an account to the Remote Desktop Users group"
+   Content: "The user has added an account to to the Remote Desktop Users group {{Count}} time(s)"
```
Comment (shainw) [inline Insights/InputEntity_Account/GroupAdditions.yaml:152]: extra **"to"**

**Review by shainw — CHANGES_REQUESTED** [review]

**Pull request merged**